Products/CMFPlone/patterns/__init__.py

```diff
@@ -242,6 +241,8 @@ def tinymce(self):
 
         image_types = settings.image_objects or []
         folder_types = settings.contains_objects or []
+
+        site_path = generator.portal_url.replace(self.request['SERVER_URL'], '')
```

Comment:
I don't know about how safe it is to use SERVER_URL. but if you just want to strip out the domain including protocol, why not using a regex for this? This regex should do it:
``.*\:\/\/((?![\/?]).)*``

This matches any protol, which is delimited by a ``:\\`` sequence. Then it matches all characters until delimited by a path delimiter ``/`` or query string delimiter ``?``, using regex negative lookahead.

E.g:
```
>>> import re
>>> re.sub(r'.*\:\/\/((?![\/?]).)*', '', 'webdav://domainname.com/abc/xyz')
'/abc/xyz'
```